pyrene/src/components/DateTimePicker/DateTimePicker.tsx

```diff
@@ -0,0 +1,303 @@
+import React, {
+  useCallback,
+  useEffect,
+  useState,
+} from 'react';
+
+import ReactDatePickerWrapper from './ReactDatePickerWrapper/ReactDatePickerWrapper';
+import DateTimeInput from './DateTimeInput/DateTimeInput';
+
+import {
+  getFutureDate, customDateFormat, convertToUTCtime,
+  convertToZoneTime, customStringToDate, getErrors,
+  hasDateError, hasTimeError, getClientTimeZone,
+} from '../../utils/DateUtils';
+
+type OnFunction = (value?: number) => void;
+
+export interface DateTimePickerProps{
+  /**
+   * Calendar is opened on input component render
+   */
+  calendarOpened?: boolean,
+  /**
+   * Close calendar on date select
+   */
+  closeOnSelect?: boolean,
```

Comment:
It seems that the `closeOnSelect` does not work as expected; it closes the picker when the `dateOnly` is selected, but if not, the picker stays open.

https://user-images.githubusercontent.com/15834481/140926805-cbcd6f0f-1a26-4890-89b7-ccaf87e7980a.mov

n.

Reply:
Thanks for explaining this part in our meeting :) I understood that it needs to have a selected time too before it closes the picker. I am resolving this comment.